tests/integration/targets/inventory_flightctl/inventory_test.yml

```
@@ -0,0 +1,192 @@
+
+        - name: Delete test fleets
+          flightctl.core.flightctl_resource:
+            <<: *connection_info
+            state: absent
+            kind: Fleet
+            name: "{{ item }}"
+          loop:
+            - 'ansible-integration-test-fleet'
+            - 'fleet-dev'
+            - 'fleet-test'
+            - 'fleet-prod'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)